Applications/Spire/Include/Spire/KeyBindings/KeyBindingsModel.hpp

```diff
@@ -0,0 +1,41 @@
+#ifndef SPIRE_KEY_BINDINGS_MODEL_HPP
+#define SPIRE_KEY_BINDINGS_MODEL_HPP
+#include "Spire/KeyBindings/OrderTask.hpp"
+#include "Spire/Ui/ComboBox.hpp"
+
+namespace Spire {
+
+  /** Defines the interface representing key bindings */
+  class KeyBindingsModel {
+    public:
+      virtual ~KeyBindingsModel() = default;
+
+      /** Returns the key bindings of the order tasks. */
+      virtual std::shared_ptr<ListModel<OrderTask>> get_order_tasks() = 0;
+
+      /** Returns the default key bindings of the order tasks. */
+      virtual const std::shared_ptr<ListModel<OrderTask>>
```

Comment:
Why is this ```const```?

Reply:
I think the default key bindings of the order tasks are immutable and the `KeyBindingsWindow` doesn't change it when the user configures the key bindings.